### PR TITLE
issue #2920: temporary fix dictation input crash on ios7.

### PR DESCRIPTION
**Pull request by boyu0**

### CODE REVIEW

**Comment by minggo** [comment]
Please add comment in codes to describe it.
May be add "XXX: Fix me ..." is better.

**Comment by boyu0** [comment]
No problem.

**Comment by minggo** [comment]
And it is better to add a issue number in comment.

File: extensions/GUI/CCEditBox/CCEditBoxImplIOS.mm [hunk 1]
```diff
@@ -414,10 +414,10 @@ - (void) textChanged
             _systemControl.textField.keyboardType = UIKeyboardTypeDecimalPad;
             break;
         case EditBox::InputMode::SINGLE_LINE:
-            _systemControl.textField.keyboardType = UIKeyboardTypeDefault;
+            _systemControl.textField.keyboardType = UIKeyboardTypeEmailAddress;
```
Comment (dumganhar) [inline extensions/GUI/CCEditBox/CCEditBoxImplIOS.mm]: Why does `SINGLE_LINE` mode use `UIKeyboardTypeEmailAddress`. It's weird.

Comment (boyu0) [inline extensions/GUI/CCEditBox/CCEditBoxImplIOS.mm]: Yes, you are right. But this is only way I can find to fix this bug for now.

Comment (dumganhar) [inline extensions/GUI/CCEditBox/CCEditBoxImplIOS.mm]: But how to input chinese if it's for email input?

Comment (boyu0) [inline extensions/GUI/CCEditBox/CCEditBoxImplIOS.mm]: I have tested that, it seems it makes no difference with default type except have more button than default at the bottom of keyboard.

Comment (dumganhar) [inline extensions/GUI/CCEditBox/CCEditBoxImplIOS.mm]: You mean that you can also input `chinese` letters in `email` mode?

**Pull request merged**